src/api/v1/register.go

```diff
@@ -77,7 +81,7 @@ func registerWithToken(c echo.Context) error {
 		return echo.NewHTTPError(http.StatusBadRequest, "bad mc uuid")
 	}
 	var userID uuid.UUID
-	err = database.DB.QueryRow("INSERT INTO users(email, password_hash, mc_uuid, discord_id) VALUES ($1, $2, $3, $4) RETURNING user_id", body.Email, hashedPassword, body.Mcuuid, body.Discord).Scan(&userID)
+	err = database.DB.QueryRow("INSERT INTO users(email, password_hash, mc_uuid, discord_id) VALUES ($1, $2, $3, $4) RETURNING user_id", body.Email, hashedPassword, body.Mcuuid, body.DiscordToken).Scan(&userID)
```

Comment:
REEEEEEEEEee dont fucking insert a user token into the discord_id column the whole point is thats unique

Reply:
good spot